070-Integrations/024-cloudflare-workers.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Cloudflare Workers with Xata
+navTitle: Cloudflare Workers
+keywords: ['cloudflare', 'integration']
+description: Use Cloudflare Workers with Xata to improve performance
+slug: integrations/cloudflare
+published: true
+---
+
+[Cloudflare](https://www.cloudflare.com/en-gb/) is a content delivery and security service that optimizes website performance and mitigates security risks for sites. [Cloudflare Workers](https://workers.cloudflare.com/) enable you to run code directly on Cloudflare's global network to reduce latency and improve performance and can be used for tasks like customizing web content, handling API requests, and modifying web traffic, for overall faster response times.
+
+The completed [Cloudflare and Xata code](https://github.com/xataio/examples/tree/main/apps/getting-started-cloudflare-workers) is available in the [Xata `examples` repo](https://github.com/xataio/examples) on GitHub.
+
+### Before you begin
+
+- Ensure you [install the Xata CLI and authenticate](https://xata.io/docs/getting-started/installation)
+
+### Create a new Cloudflare app
+
+Choose the "Hello World" example and accept the default prompt options.
+
+```sh
+npm create cloudflare@latest getting-started-cloudflare-workers
+```
+
+After the command has completed, navigate to the `getting-started-cloudflare-workers` directory, and run the application locally:
+
+```sh
+npm run wrangler dev
+```
+
+By default, the application runs on `http://localhost:8787`. When visiting this URL "Hello World" should be shown in your browser.
+
+### Deploy your app remotely
+
+```sh
+npx wrangler deploy
+```
+
+Preview your Worker at `<YOUR_WORKER>.<YOUR_SUBDOMAIN>.workers.dev.` When you visit this URL in your browser you should see "Hello World" returned.
+
+### Install Xata Client
+
+Inside your project root, run the following:
+
+```sh
+xata init
+```
+
+Accept the defaults. By the end you should have a `.env` file generated for you within your app folder. This file will contain the correct credentials to access your database from your Cloudflare worker.
+
+```bash title=".env"
+XATA_API_KEY=YOUR_API_KEY_HERE
```

Comment:
Noob question: How do users know where to go to get this key?

Reply:
Ahh we've expected the user to go through the "before you begin" bit [here](https://xata-docs-pr-182.vercel.app/docs/integrations/cloudflare#before-you-begin), and at the bottom describe how to get the API key. 🤔